index.html

```diff
@@ -3001,6 +2999,38 @@ <h3>
 
         </section>
 
+        <section class="normative">
+            <h3>
+                Add Index
+            </h3>
+
+            <p>
+  Encrypted Documents can have embedded key-value attribute pairs, as seen in <a href="#blinded-document-attributes"></a>.
+  To speed up searches in vaults containing many Encrypted Documents, indexes can be created based on attribute keys.
+            </p>
+
+
+            <p>
+  The example below shows how to create indexes in a vault.
+            </p>
+
+            <pre class="example highlight" title="add indexes to data vault">
+  POST https://example.com/edvs/z4sRgBJJLnYy/index HTTP/1.1
+  Host: example.com
+  Content-Type: application/json
+  Accept: application/json, text/plain, */*
+  Accept-Encoding: gzip, deflate
+
+  {
+    "operation": "add",
+    "attributeKeys":["DUQaxPtSLtd8L3WBAIkJ4DiVJeqoF6bdnhR7lSaPloZ", "AarngVIZLl0kIp2xEHUH5o5uVc-470roQaOIbqMUD7DFQQypWQ=="]
```

Comment:
```suggestion
    "attributeKeys":["DUQaxPtSLtd8L3WBAIkJ4DiVJeqoF6bdnhR7lSaPloZ", "AarngVIZLl0kIp2xEHUH5o5uVc-470roQaOIbqMUD7DFQQypWQ=="]
```

is this an attribute key or an attribute name-value pair?

Reply:
It's a set of attribute keys.